src/main/java/org/jenkinsci/plugins/casc/integrations/GlobalMatrixAuthorizationStrategyConfigurator.java

```diff
@@ -0,0 +1,59 @@
+package org.jenkinsci.plugins.casc.integrations;
+
+import hudson.Extension;
+import hudson.security.GlobalMatrixAuthorizationStrategy;
+import hudson.security.Permission;
+import org.jenkinsci.plugins.casc.Attribute;
+import org.jenkinsci.plugins.casc.Configurator;
+import org.jenkinsci.plugins.casc.RootElementConfigurator;
+
+import java.util.*;
+
+/**
+ * Example
+ *
+ * jenkins:
```

Comment:
better to just reference the test YAML IMHO

Reply:
Added the example to the demos instead